nifpga/fixedpointhelper.py

```diff
@@ -0,0 +1,59 @@
+from warnings import warn
+
+
+def bin_to_int(binary_string):
+    if binary_string == '':
+        return 0
+    else:
+        return int(binary_string, 2)
+
+
+def to_bin(value):
+    """
+    This class uses the built in bin() function to do the conversion,
+    but we remove the leading '0b' or '-0b' identifier to keep the logic
+    simplified.
+    """
+    binary_representation = bin(value)
+    if binary_representation[0] == '-':
+        return binary_representation[3:]
+    else:
+        return binary_representation[2:]
+
+
+def twos_compliment(binary_string):
```

Comment:
How do you compliment a 2? "Hey! Nice serif!" 
You're looking for twos_complement, as in "complete the power of 2".

Reply:
Removed in "Removing string based implementation commit"